frontend/src/components/CourseButtonCard /CourseButtonCard.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import QuickAddCartButton from 'components/QuickAddCartButton';
+import { addTab } from 'reducers/courseTabsSlice';
+import S from './styles';
+
+type Props = {
+  courseCode: string
+  title: string
+  planned?: boolean
+};
+
+const CourseButtonCard = ({ courseCode, title, planned }: Props) => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const handleCourseLink = (e: React.MouseEvent<HTMLElement>): void => {
+    e.stopPropagation();
```

Comment:
umm, cant you use `type='button'` on the button instead to avoid this? Or am i smoking something?

Reply:
hehe so uh, I like to follow the convention of existing code ;)

Reply:
@xincczn I dont think you would have wanted to join circles if me and @gtangelo followed the code conventions of existing code 😆